docs/versioning.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Generator Version vs Template Version"
+weight: 60
+---
```

Comment:
This doc is about **the generator version vs the template version** but I found most of what I explained is about the generator version, not the template version. This is one thing I think can be improved in this doc. Maybe you can give me some pointers.